api/tests/src/api/opportunities_v1/test_opportunity_route_search.py

```diff
@@ -710,6 +710,69 @@ def test_search_filters_200(
     ):
         call_search_and_validate(client, api_auth_token, search_request, expected_results)
 
+    @pytest.mark.parametrize(
+        "search_request, expected_status_code",
+        [
+            # Post Date
+            (get_search_request(post_date={"start_date": None}), 200),
+            (get_search_request(post_date={"end_date": None}), 200),
+            (get_search_request(post_date={"start_date": "2020-01-01"}), 200),
+            (get_search_request(post_date={"end_date": "2020-02-01"}), 200),
+            (get_search_request(post_date={"start_date": None, "end_date": None}), 200),
+            (get_search_request(post_date={"start_date": "2020-01-01", "end_date": None}), 200),
+            (get_search_request(post_date={"start_date": None, "end_date": "2020-02-01"}), 200),
+            (
+                get_search_request(
+                    post_date={"start_date": "2020-01-01", "end_date": "2020-02-01"}
+                ),
+                200,
+            ),
+            (get_search_request(post_date={"start_date": "I am not a date"}), 422),
+            (get_search_request(post_date={"start_date": "123-456-789"}), 422),
+            (get_search_request(post_date={"start_date": "5"}), 422),
+            (get_search_request(post_date={"start_date": 5}), 422),
+            (get_search_request(post_date={"end_date": "I am not a date"}), 422),
+            (get_search_request(post_date={"end_date": "123-456-789"}), 422),
+            (get_search_request(post_date={"end_date": "5"}), 422),
+            (get_search_request(post_date={"end_date": 5}), 422),
```

Comment:
Just because this is a lot of scenarios in one test, I might recommend splitting this into two "smaller" tests. One for the 200 scenarios, and one for the 422 scenarios.

Just name them something like `test_search_validate_date_filters_200` and `test_search_validate_date_filters_422`.

Reply:
Done :)